.github/workflows/main.yml

```diff
@@ -56,6 +56,12 @@ jobs:
           name: python-artifacts
           path: .
 
+      - name: Upload helm artifacts
+        uses: actions/upload-artifacts@v4
```

Comment:
There is a typo in the action name `actions/upload-artifacts@v4`. It should be `actions/upload-artifact@v4` to match the correct GitHub Action for uploading artifacts.